sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1037,6 +1041,11 @@ public partial class BlockBlobClient : Azure.Storage.Blobs.Specialized.BlobBaseC
         public new Azure.Storage.Blobs.Specialized.BlockBlobClient WithSnapshot(string snapshot) { throw null; }
         protected sealed override Azure.Storage.Blobs.Specialized.BlobBaseClient WithSnapshotCore(string snapshot) { throw null; }
     }
+    public partial class ExtendedBlobClientOptions : Azure.Storage.Blobs.BlobClientOptions
```

Comment:
`Extended` is probably not the best prefix because a lot of folks have a visceral reaction to it.  Could we call this `SpecializedBlobClientOptions` to match the `Specialized` namespace and terminology we've been building?